Update build-w-o-db: Partial build / remove example (#727)
shopware/docs: guides/hosting/installation-updates/deployments/build-w-o-db.md

By default, Shopware builds the Administration without extensions if there is no database connection. To include the extensions without a database, you will need to use the `ComposerPluginLoader`. This determines the used plugins by looking up the installed project dependencies. To get this working the plugin needs to be required in the system using `composer req [package/name]`.

There is a file `bin/ci` which uses the `ComposerPluginLoader` and can be used instead of `bin/console`.

> Using this, we can dump the plugins for the Administration with the new file without a database with the command `bin/ci bundle:dump`. It is recommended to call `bin/ci` instead of `bin/console` in the `bin/*.js` scripts, which can be achieved by setting the environment variable `CI=1`.

Reviewer: Please use "you" instead of "we" :) - https://github.com/shopware/docs/pull/701/files#diff-e6bbdeb8f95aaa78bd37d6385a694ea62f498ad1fb4b5ca95e0fb5840008be86R41

Author: changed - thanks for the remark :-)

> Using this, we can dump the plugins for the Administration with the new file without a database with the command `bin/ci bundle:dump`. It is recommended to call `bin/ci` instead of `bin/console` in the `bin/*.js` scripts, which can be achieved by setting the environment variable `CI=1`.

Reviewer: ```suggestion
Using this, we can dump the plugins for the Administration with the new file without a database using the command `bin/ci bundle:dump`. It is recommended to call `bin/ci` instead of `bin/console` in the `bin/*.js` scripts, which can be achieved by setting the environment variable `CI=1`.
```

 ```

 This will force the theme compiler to use the static dumped file instead of looking into the database.

### Partially compiling the Storefront

> You can also build just the Javascript bundle using `CI=1 SHOPWARE_SKIP_THEME_COMPILE=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true bin/build-storefront.sh'` (without the need for the above loader) in your CI and later run `bin/console theme:dump` on your production system, when the database is available. This will happen anyways, if theme variables are changed via the admin panel.

Reviewer: ```suggestion
You can also build just the Javascript bundle using `CI=1 SHOPWARE_SKIP_THEME_COMPILE=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true bin/build-storefront.sh` (without the need for the above loader) in your CI. After that, run `bin/console theme:dump` on your production system, when the database is available. This will happen automatically, if theme variables are changed via the admin panel.
```
